tests/evaluate/test_default_evaluator.py

```diff
@@ -3409,6 +3429,9 @@ def fn3(X):
         "recall_at_3/mean": 0.5,
         "recall_at_3/p90": 0.5,
         "recall_at_3/variance": 0.0,
+        "ndcg_at_3/mean": 0.6131471927654585,
+        "ndcg_at_3/p90": 0.6131471927654585,
+        "ndcg_at_3/variance": 0.0,
```

Comment:
I think we may want to update `test_evaluate_retriever_builtin_metrics_no_model_type` as well